packages/augur-ui/src/modules/modal/containers/modal-test-bet.ts

```diff
@@ -20,13 +20,14 @@ const mapDispatchToProps = (dispatch: ThunkDispatch<void, any, Action>) => ({
 });
 
 const mergeProps = (sP: any, dP: any, oP: any) => ({
-  largeHeader: 'Learn how to place a bet on Augur ',
+  largeHeader: '2. Run a test bet',
+  smallHeader: 'Learn how to place a bet using Augur Trade',
   testBet: true,
-  mediumHeader: 'Place a test bet',
+  progress: 3,
```

Comment:
can you change `progress` to something else. b/c its it's own thing. I don't want to have progressLabel and progress in the codebase since they are different concepts.